docker-compose.e2e-override-server.yml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# CERN Analysis Preservation is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+version: "2.3"
+services:
+  app:
+    image: gitlab-registry.cern.ch/pshandil/cap-server/app:build-e2e # To change
```

Comment:
Do wee need those in the `Dockerfile`

https://gitlab.cern.ch/pshandil/cap-server/-/blob/master/Dockerfile#L76-L77

Reply:
I removed it since it was testing.